rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -423,15 +441,39 @@ s32 cellGemForceRGB(u32 gem_num, float r, float g, float b)
 	return CELL_OK;
 }
 
-s32 cellGemGetAccelerometerPositionInDevice()
+s32 cellGemGetAccelerometerPositionInDevice(u32 gem_num, vm::ptr<f32[4]> pos)
```

Comment:
should this be a vm::pptr ?

Reply:
Oh yeah, I think it should. Maybe there are overloads for such arrays in `vm::ptr` but better safe than sorry.

Reply:
Actually no, isn't this just a pointer to 4 floats? Maybe it's fine as is.
Just leave whatever and I'll verify it at some point.

Reply:
it should be different from the other occurance

Reply:
since one is a getter and one is a setter